test/distributed/pipelining/test_schedule_multiproc.py

```diff
@@ -757,7 +753,6 @@ def dw_runner():
                 stage_idx,
                 n_stages,
                 self.device,
-                input_args=input_args,
```

Comment:
no, i thought about splitting it up that way but since I marked it as deprecated I decided to go all in on covering the shape inference path instead.  I can add one back in though, its a good point.

Reply:
I decided to add coverage for the case where we pass both input and output, since that's supported as an override to disable shape inference.  I left the input_args only case untested.  I wonder if we can just move quickly to stop supporting that path? Do we need a deprecation cycle or should we just flip it? I can put up another PR for this change and we can discuss there.